README.md

```diff
@@ -1 +1,113 @@
-# entando-mysql
\ No newline at end of file
+MySQL 8.0 SQL Database Server container image
+=============================================
+
+This container image includes MySQL 8.0 SQL database server for  general usage.
+Users can choose between RHEL8 and RockyLinux8 based images.
+
+Description
+-----------
+
+This container image provides a containerized packaging of the MySQL mysqld daemon
+and client application. The mysqld server daemon accepts connections from clients
+and provides access to content from MySQL databases on behalf of the clients.
+
+Usage
+-----
+
+If you want to set only the mandatory environment variables and not store
+the database in a host directory, execute the following command:
+
+```
+$ podman run -d --name mysql_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306 (Image:tag)
+```
+
+This will create a container named `mysql_database` running MySQL with database
+`db` and user with credentials `user:pass`. Port 3306 will be exposed and mapped
+to the host. If you want your database to be persistent across container executions,
+also add a `-v /host/db/path:/var/lib/mysql/data` argument. This will be the MySQL
+data directory.
+
+If the database directory is not initialized, the entrypoint script will first
+run [`mysql_install_db`](https://dev.mysql.com/doc/refman/en/mysql-install-db.html)
+and setup necessary database users and passwords. After the database is initialized,
+or if it was already present, `mysqld` is executed and will run as PID 1. You can
+ stop the detached container by running `podman stop mysql_database`.
```

Comment:
I would add an info here: "Please note that the container does not handle SIGINT (CTRL+C) signal, only SIGTERM (Ctrl+\) as the first can bring issues with data integrity.